docs/source/models/gbd2017/causes/ischemic_heart_disease/ischemic_heart_disease.rst

```diff
@@ -0,0 +1,68 @@
+.. _2017_cause_ischemic_heart_disease:
+
+===================
+Ischemic Heart Disease
+===================
+
+Disease Description
+-------------------
+
+.. todo::
+
+   Add more information and references. In particular, find data about global prevalence and relation to disease fatal and non-fatal description.
+
+
+
+
+
+Modeling Ischemic Heart Disease in GBD 2017
+-------------------------------------------
+
+Myocardial infarction (MI) in GBD 2017
+++++++++++++++++++++++++++++++++++++++
+
+
+Modeling Strategy for Ischemic Heart Disease
+++++++++++++++++++++++++++++++++++++++++++++
+
+Cause Model Diagram
+--------------------------
+.. image:: ihd_states.png
+  :width: 150
+
+Transitions Diagram
+--------------------------
+.. image:: ihd_transitions.png
+  :width: 200
+
+Model Assumptions and Limitations
```

Comment:
I vote for moving this section after the Data Description section, so that the model diagram and data tables are next to each other.

Reply:
I made this update. Thanks!